models/productModel.mjs

```diff
@@ -0,0 +1,74 @@
+import mongoose from 'mongoose';
+
+const ProductSchema = mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+    },
+    brand: {
+        type: String,
+        required: true,
+    },
+    price: {
+        type: String,
+        required: true,
+    },
+    options: [],
+    reviews: [],
+    created_on: {
+        type: Date,
+        default: Date.now
+    }
+});
+
+const Product = mongoose.model('Products', ProductSchema);
+
+Product.create({
```

Comment:
Export into mock files and add array of mock products, please
Do the same way with user if need, please

Reply:
Done.